src/rajAJun20/MaximumDifferIndexValue.java

```diff
@@ -0,0 +1,27 @@
+package rajAJun20;
+
+public class MaximumDifferIndexValue {
+	
+	void maxValueOfDifferIndex(int[] arr1 , int[] arr2) {
+		if(arr1.length == arr2.length) {
+			for(int index = 0 ; index < arr1.length ; index++) {
+				if(arr1[index] != arr2[index]) {
+					System.out.println("Numbers are not matching at index : " + index);
+				if(arr1[index] > arr2[index]) 
```

Comment:
Again this is wrong. Here you should write second if inside first if.

Reply:
Done